### PR TITLE
add italian translation for README.md

### PR DESCRIPTION
**Pull request by Francesca-Belfiore**
### Changelog

__Component__ : server / cli / console / build 

__Type__: bugfix / feature / enhancement 

__Product__: community-edition

#### Short Changelog

Added the italian translation for the Readme.

#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [ ] No
- [ ] Yes
  - [ ] Updated docs with SQL for downgrading the catalog 

#### Metadata

Does this PR add a new Metadata feature?
- [ ] No
- [ ] Yes
  - Does `run_sql` auto manages the new metadata through schema diffing?
    - [ ] Yes
    - [ ] Not required
  - Does `run_sql` auto manages the definitions of metadata on renaming?
    - [ ] Yes
    - [ ] Not required
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?
    - [ ] Yes
    - [ ] Not required

#### GraphQL
- [ ] No new GraphQL schema is generated
- [ ] New GraphQL schema is being generated:
   - [ ] New types and typenames are correlated
   
   

#### Breaking changes

- [ ] No Breaking changes
- [ ] There are breaking changes:

  1. Metadata API

     Existing `query` types:
     - [ ] Modify `args` payload which is not backward compatible
     - [ ] Behavioural change of the API
     - [ ] Change in response `JSON` schema
     - [ ] Change in error code
     

  2. GraphQL API

     Schema Generation:
     
     
     - [ ] Change in any `NamedType`
     - [ ] Change in table field names
     

     Schema Resolve:-
     
     - [x] Change in treatment of `null` value for any input fields 
     

  3. Logging

     - [ ] Log `JSON` schema has changed
     - [ ] Log `type` names have changed

### CODE REVIEW

File: translations/README.italian.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+[https://hasura.io/newsletter/](https://hasura.io/newsletter/)
+
+## Contribuire
+
+Dai un'occhiata alla nostra [guida per i contributi](CONTRIBUTING.md) per ulteriori dettagli.
+
+## Risorse del Marchio
+
+Le risorse del marchio Hasura (loghi, la mascotte di Hasura, i badge "powered by", etc.) possono
+essere trovate nella cartella [assets/brand](../assets/brand). Sentiti libero di userle nella tua
```
Comment (lucarestagno) [inline translations/README.italian.md:184]: **typo**: `userle` -> `usarle`

File: translations/README.italian.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+- [Spanish 🇲🇽](../translations/README.mx_spanish.md)(:pray: [@ferdox2](https://github.com/ferdox2))
+- [Indonesian :indonesia:](../translations/README.indonesian.md) (:pray: [@anwari666](https://github.com/anwari666))
+- [Brazilian Portuguese :brazil:](../translations/README.portuguese_br.md) (:pray: [@rubensmp](https://github.com/rubensmp))
+- [German 🇩🇪](../translations/README.german.md) (:pray: [@FynnGrandke](https://github.com/FynnGrandke))
+- [Chinese :cn:](../translations/README.chinese.md) (:pray: [@jagreetdg](https://github.com/jagreetdg) & [@johnbanq](https://github.com/johnbanq))
+- [Turkish :tr:](../translations/README.turkish.md) (:pray: [@berat](https://github.com/berat))
+- [Korean :kr:](../translations/README.korean.md) (:pray: [@라스크](https://github.com/laskdjlaskdj12))
+- [Italian :it:](../translations/README.italian.md) (:pray: [@befire](https://github.com/francesca-belfiore))
+
+Le traduzioni degli altri file possono essere trovate [qui](translations).
```
Comment (lucarestagno) [inline translations/README.italian.md:231]: **suggestion**: I would use `Le traduzioni di altri file` 🙌

File: translations/README.italian.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
+# Hasura GraphQL Engine
+
+[![Latest release](https://img.shields.io/github/v/release/hasura/graphql-engine)](https://github.com/hasura/graphql-engine/releases/latest)
+[![Docs](https://img.shields.io/badge/docs-v2.x-brightgreen.svg?style=flat)](https://hasura.io/docs)
+
+<a href="https://discord.gg/vBPpJkS"><img src="https://img.shields.io/badge/chat-discord-brightgreen.svg?logo=discord&style=flat"></a>
+<a href="https://twitter.com/intent/follow?screen_name=HasuraHQ"><img src="https://img.shields.io/badge/Follow-HasuraHQ-blue.svg?style=flat&logo=twitter"></a>
+<a href="https://hasura.io/newsletter/"><img src="https://img.shields.io/badge/newsletter-subscribe-yellow.svg?style=flat"></a>
+
+Hasura è un prodotto open source che accelera lo sviluppo API di 10 volte fornendoti all'istante [GraphQL](https://hasura.io/graphql/) o API REST con autorizzazione integrata sui tuoi dati.
```
Comment (lucarestagno) [inline translations/README.italian.md:10]: **suggestion**: I would use `fornendoti all'istante API GraphQL or REST`

**suggestion**:  I would remove `sui tuoi dati`

File: translations/README.italian.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+
+------------------
+
+![Hasura GraphQL Engine Realtime Demo](../assets/realtime.gif)
+
+-------------------
+
+## Features
+
+* **Crea queries potenti**: Filtro, paginazione, ricerca pattern, inserimento in bulk, aggiornamento ed eliminazione mutazioni integrati
```
Comment (lucarestagno) [inline translations/README.italian.md:26]: **rephrase**: I would use

`Filtri, paginazione, ricerca di pattern, e mutazioni di inserimento, aggiornamento ed eliminazione, anche in bulk`

File: translations/README.italian.md [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+* **Lavora con live database esistenti**: Punta a un database esistente per ottenere istantaneamente una GraphQL API pronta all'uso
+* **In tempo reale**: Converti qualsiasi query GraphQL in una query live usando le sottoscrizioni
+* **Merge di schemi remoti**: Accedi a schemi GraphQL personalizzati per la business logic tramite un singolo endpoint GraphQL Engine. [**Leggi di più**](remote-schemas.md).
+* **Estendibile con le Azioni**: Scrivi API REST per estendere lo schema di Hasura con una business logic personalizzata.
+* **Attiva webhook o funzioni serverless**: Inserisci/aggiorna/elimina eventi su Postgres ([leggi di più](event-triggers.md))
```
Comment (lucarestagno) [inline translations/README.italian.md:31]: **todo**: I would use `Eventi di inserimento/aggiornamento/eliminazione su Postgres`

File: translations/README.italian.md [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+* **Trigger programmati**: Esegui la business logic personalizzata in momenti specifici utilizzando una configurazione cron o un singolo evento.
+* **Controllo degli accessi preciso**: Controllo accessi dinamico che si integra con il tuo sistema di autenticazione (es: auth0, firebase-auth)
+* **Admin UI & Migrazioni**: Migrazioni dell'UI dell'amministratore e delgli schemi ispirate a Rails.
```
Comment (lucarestagno) [inline translations/README.italian.md:34]: **todo**: `Interfaccia utente di amministrazione & migrazioni degli schemi ispirate a Rails`

File: translations/README.italian.md [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+
+- [Guida introduttiva:](#quickstart)
+    - [Deployment one-click su Hasura Cloud](#one-click-deployment-on-hasura-cloud)
+    - [Altre opzioni di deployment one-click](#other-one-click-deployment-options)
+    - [Altri metodi di deployment](#other-deployment-methods)
+- [Architettura](#architecture)
+- [Strumenti client-side](#client-side-tooling)
+- [Aggiungere la business logic](#add-business-logic)
+    - [Schemi remoti](#remote-schemas)
+    - [Attiva webhook su eventi del database](#trigger-webhooks-on-database-events)
```
Comment (lucarestagno) [inline translations/README.italian.md:51]: **suggestion**: I would use `Invoca` instead of `Attiva`

File: translations/README.italian.md [hunk 8]
```diff
@@ -0,0 +1,231 @@
...
+
+GraphQL Engine fornisce metodi semplici, scalabili e performanti per aggiungere business logic personalizzate al tuo back-end:
+
+### Schemi remoti
+
+Aggiungi risolutori personalizzati in uno schema remoto in aggiunta allo schema GraphQL basato su database di Hasura. Ideale per casi d'uso come l'implementazione di un'API di pagamento o l'esecuzione di query su dati non presenti nel tuo database - [leggi di più](remote-schemas.md).
+
+### Azioni
+
+Le azioni sono un modo per estendere lo schema di Hasura con una business logic personalizzata utilizzando query e mutazioni ad hoc. È possibile aggiungere azioni ad Hasura per gestire vari casi d'uso come la convalida dei dati, l'arricchimento dei dati da fonti esterne e qualsiasi altra buniness logic complessa - [leggi di più](https://hasura.io/docs/latest/graphql/core/actions/index.html)
```
Comment (lucarestagno) [inline translations/README.italian.md:121]: **typo**: `buniness` -> `business` 😉

**Review by lucarestagno — CHANGES_REQUESTED** [review]
Thanks for your contribution Francesca!

You did a great job 🙌

I left some comments, would you mind looking at them?

Grazie 😉

**Comment by Francesca-Belfiore** [comment]
Ciao! Thank you for the review! I will apply the corrections as soon as possible 😉

**Review by lucarestagno — APPROVED** [review]
Looks good to me! Thanks Francesca! 🙌

**Comment by Francesca-Belfiore** [comment]
You're welcome!
Oh, I saw you added the hacktoberfest label but since this repository is not participating the correct label to make this pr valid should be hacktoberfest-accepted. Could you please change it? Thank you in advance!

**Comment by Stefmore02** [comment]
Hey! @Francesca-Belfiore 

We truly appreciate your contribution to Hasura OSS during Hacktoberfest 2022! We'd love to send you some Hasura swag as a token of appreciation. Could you please send me an email to [estefani@hasura.io](mailto:estefani@hasura.io) with a reference of this PR so I can send you the swag form? ✨

**Comment by Francesca-Belfiore** [comment]
Sure thanks, it would be a pleasure!

**Comment by Stefmore02** [comment]
Email received, I also forwarded the swag form for you to fill out. Thank you! 👍 ⭐

**Pull request closed without merging**